src/App.php

```diff
@@ -265,13 +265,21 @@ protected function configureRoutes( Slim $slim ) {
 				$middleware['inject-user'], $middleware['set-environment'],
 			function () use ( $slim, $middleware, $routeConditions ) {
 				$slim->get( '', function() use ( $slim ) {
+					// See if we have a cookie indicating last version used
+					if ( isset( $_COOKIE['copypatrolLang'] ) ) {
```

Comment:
I looked into that earlier. Slim only has session cookies whereas I thought we wanted ours to persist longer. I realize now I forgot to give it an expiry date. I'm thinking 30 days should be good.